modules/etcd-encryption-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * security/encrypting-etcd.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: CONCEPT
+[id="etcd-encryption-types_{context}"]
+= Supported encryption types
+
+The following encryption types are supported for encrypting etcd data in {product-title}:
+
+AES-CBC:: Uses AES-CBC with PKCS#7 padding and a 32 byte key to perform the encryption. The encryption keys are rotated on a weekly basis.
+
+AES-GCM:: Uses AES-GCM with a random nonce and a 32 byte key to perform the encryption. The encryption keys are rotated after every 200,000 uses of the keys.
```

Comment:
Contrarily to what I initially told you, we going to rotate AES-GCM keys on a weekly basis as well.

I am working on an upstream effort to add the reasoning behind rotation periods to the official doc.
It might be worth pointing to it once it is live. For now it is still a work in progress that can be viewed in https://deploy-preview-39747--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/encrypt-data/#aesgcm

Reply:
Updated!